### PR TITLE
feat(@desktop/wallet): Using Floating Buttons in the Send Modal

### LINKED ISSUE #6228: Using Floating Buttons in the Send Modal
Use the new floating buttons in the Modal in the Send Modal. Wasn't done as the widget wasn't available then

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #6228

### What does the PR do

Uses the new floating buttons selector in the SendModal

### Affected areas

Wallet

### CODE REVIEW

**Review by alaibe — APPROVED** [review]

**Review by anastasiyaig — APPROVED** [review]

File: ui/imports/shared/popups/SendModal.qml [hunk 1]
```diff
@@ -74,27 +74,43 @@ StatusModal {
...
     })
 
+    QtObject {
+        id: d
+        property string maxFiatBalance: Utils.stripTrailingZeros(parseFloat(assetSelector.selectedAsset.totalBalance).toFixed(4))
+        //% "Please enter a valid amount"
+        property string sendAmountInputErrorMessage: qsTr("Please enter a valid amount")
+        //% "Max:"
+        property string maxString: qsTr("Max: ")
+        property bool isReady: amountToSendInput.valid && !amountToSendInput.pending && recipientSelector.isValid && !recipientSelector.isPending
```
Comment (caybro) [inline ui/imports/shared/popups/SendModal.qml:84]: `readonly`?

File: ui/imports/shared/popups/SendModal.qml [hunk 2]
```diff
@@ -148,15 +151,151 @@ StatusModal {
...
+                ColumnLayout {
+                    id: assetAndAmmountSelector
+                    anchors.top: parent.top
+                    anchors.right: parent.right
+                    anchors.left: parent.left
+                    RowLayout {
+                        spacing: 16
+                        StatusBaseText {
+                            //% "Send"
+                            text: qsTrId("command-button-send")
```
Comment (caybro) [inline ui/imports/shared/popups/SendModal.qml:163]: `qsTr("Send")`

Comment (Khushboo-dev-cpp) [reply ui/imports/shared/popups/SendModal.qml:163]: this is already added as a translation can be reused here?

File: ui/imports/shared/popups/SendModal.qml [hunk 3]
```diff
@@ -148,15 +151,151 @@ StatusModal {
                 contentHeight: addressSelector.height + networkSelector.height + gasSelector.height + gasValidator.height
                 clip: true
 
+                ColumnLayout {
```
[the remark this answers is not in the document]
Comment (Khushboo-dev-cpp) [reply ui/imports/shared/popups/SendModal.qml:154]: Hey Igor, Can you explain why its needs to be a Column within a ScrollView?

Also did not understand the second part.

**Comment by Khushboo-dev-cpp** [comment]
> The important comment is about `Popup` sizing. The rest are minor changes.
> 
> Also, I noted to UI bugs:
> 
> 1. When hovering `Use custom` priority button, it's background overlaps `High` floating button.
> 2. It's possible to have no priority selected by deselecting one. It should probably  behave as `RadioButton`.
> 
> image

Added a linkt to these two bugs here https://github.com/status-im/status-desktop/issues/6268. Will do it under that

File: ui/imports/shared/popups/SendModal.qml [hunk 4]
```diff
@@ -74,27 +74,43 @@ StatusModal {
         }
     })
 
+    QtObject {
+        id: d
+        property string maxFiatBalance: Utils.stripTrailingZeros(parseFloat(assetSelector.selectedAsset.totalBalance).toFixed(4))
+        //% "Please enter a valid amount"
+        property string sendAmountInputErrorMessage: qsTr("Please enter a valid amount")
+        //% "Max:"
+        property string maxString: qsTr("Max: ")
```
[the remark this answers is not in the document]
Comment (Khushboo-dev-cpp) [reply ui/imports/shared/popups/SendModal.qml]: Done, makes sense!

**Comment by Khushboo-dev-cpp** [comment]
All done check when you can @caybro @igor-sirotin

**Review by caybro — APPROVED** [review]

File: ui/imports/shared/popups/SendModal.qml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Khushboo-dev-cpp) [reply ui/imports/shared/popups/SendModal.qml]: will double check this in the task I am working on now.

**Pull request merged**